scripts/vm/network/security_group.py

```diff
@@ -35,11 +33,6 @@
 logpath = "/var/run/cloud/"        # FIXME: Logs should reside in /var/log/cloud
```

Comment:
Since you are refactoring, should we perhaps fix the logpath as well?

Reply:
I am not sure what the implications might be.

People might be parsing the files at that location, but also, if they are in /var/run/cloud they will be gone after a reboot (which is good!)

Reply:
5.0.0 sounds like a good plan.